automation/notebooks-table-data.csv

```diff
@@ -1,27 +1,36 @@
-OpenAI Clip PyTorch Classification, how-to-use-openai-clip-classification.ipynb, https://blog.roboflow.com/how-to-use-openai-clip, https://www.youtube.com/watch?v=8o701AEoZ8I, https://github.com/openai/CLIP, False, models
-YOLOv4-tiny Darknet Object Detection, train-yolov4-tiny-object-detection-on-custom-data.ipynb, https://blog.roboflow.ai/train-yolov4-tiny-on-custom-data-lighting-fast-detection, https://www.youtube.com/watch?v=NTnZgLsk_DA, https://github.com/roboflow-ai/darknet, False, models
-RF100 Object Detection Model Benchmarking, how-to-use-rf100.ipynb, https://blog.roboflow.com/roboflow-100, https://youtu.be/jIgZMr-PBMo, https://github.com/roboflow-ai/roboflow-100-benchmark, False, skills
-Detect and Count Objects in Polygon Zone with YOLOv5 / YOLOv8 / Detectron2 + Supervision, how-to-detect-and-count-objects-in-polygon-zone.ipynb, , https://youtu.be/l_kf9CfZ_8M, https://github.com/roboflow/supervision, True, skills
-Track and Count Vehicles with YOLOv8 + ByteTRACK + Supervision, how-to-track-and-count-vehicles-with-yolov8.ipynb, https://blog.roboflow.com/yolov8-tracking-and-counting/, https://youtu.be/OS5qI9YBkfk, https://github.com/roboflow/supervision, True, skills
-Football Players Tracking with YOLOv5 + ByteTRACK, how-to-track-football-players.ipynb, https://blog.roboflow.com/track-football-players, https://youtu.be/QCG8QMhga9k, https://github.com/ifzhang/ByteTrack, True, skills
-Create Segmentation Masks with Roboflow, how-to-generate-segmentation-mask-with-roboflow.ipynb, https://blog.roboflow.com/how-to-create-segmentation-masks-with-roboflow, , , False, skills
\ No newline at end of file
+display_name, notebook_name, roboflow_blogpost_path, youtube_video_path, github_repository_path, arxiv_index, should_open_in_sagemaker_labs, readme_section
+YOLO-NAS Object Detection, train-yolo-nas-on-custom-dataset.ipynb, https://blog.roboflow.com/yolo-nas-how-to-train-on-custom-dataset, https://youtu.be/V-H3eoPUnA8, https://github.com/Deci-AI/super-gradients/blob/master/YOLONAS.md, , True, models
+Segment Anything Model, how-to-segment-anything-with-sam.ipynb, https://blog.roboflow.com/how-to-use-segment-anything-model-sam, https://youtu.be/D-D6ZmadzPE, https://github.com/facebookresearch/segment-anything, 2304.02643, True, models
+Zero-Shot Object Detection with Grounding DINO, zero-shot-object-detection-with-grounding-dino.ipynb, https://blog.roboflow.com/grounding-dino-zero-shot-object-detection, https://youtu.be/cMa77r3YrDk, https://github.com/IDEA-Research/GroundingDINO, 2303.05499, True, models
+DETR Transformer Object Detection, train-huggingface-detr-on-custom-dataset.ipynb, , https://youtu.be/AM8D4j9KoaU, https://github.com/facebookresearch/detr, 2005.12872, True, models
+DINOv2 Image Retrieval, dinov2-image-retrieval.ipynb, , , https://github.com/facebookresearch/dinov2/, False, models
```

Comment:
Let's add `DINOv2: Learning Robust Visual Features without Supervision` arXiv index `2304.07193` here. It will display a button leading to a paper related to DINOv2 in README.md.

Reply:
Also, I would move it to the `skills` section, not the `models` section. In `models` we usually show people how to remain.